### PR TITLE
Update OSX folder icons to Vircadia & OSX build docs

### PR DESCRIPTION
**Pull request by kasenvr**
(p.s. I didn't actually check to see if this works, but I assume it did!)

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-e89cf87.tar_9f810700.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-e89cf87_9f810700.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-e89cf87.tar_9f810700.gz

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-e89cf87_9f810700.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-e89cf87_9f810700.exe

**Comment by kasenvr** [comment]
image
Can confirm, she did work. :)

File: BUILD_OSX.md [hunk 1]
```diff
@@ -28,18 +30,20 @@ For OpenSSL installed via homebrew, set OPENSSL_ROOT_DIR via
...
 ### Xcode
 
-If Xcode is your editor of choice, you can ask CMake to generate Xcode project files instead of Unix Makefiles. You will need to select the Xcode installation in the terminal first if you have not done so already.
+You can ask CMake to generate Xcode project files instead of Unix Makefiles using the `-G Xcode` parameter after CMake. You will need to select the Xcode installation in the terminal first if you have not done so already.
     
     sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
-
-    cmake .. -G Xcode
+    
+    sudo cmake ../ -DCMAKE_OSX_SYSROOT="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk" -DCMAKE_OSX_DEPLOYMENT_TARGET=10.12 -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl -DOSX_SDK=10.12  ..
```
Comment (yuriymacdev) [inline BUILD_OSX.md:37]: I think sudo here is redundant

File: BUILD_OSX.md [hunk 2]
```diff
@@ -28,18 +30,20 @@ For OpenSSL installed via homebrew, set OPENSSL_ROOT_DIR via
...
 
 If `cmake` complains about Python 3 being missing, you may need to update your CMake binary with command `brew upgrade cmake`, or by downloading and running the latest CMake installer, depending on how you originally instaled CMake 
 
-After running cmake, you will have the make files or Xcode project file necessary to build all of the components. Open the hifi.xcodeproj file, choose ALL_BUILD from the Product > Scheme menu (or target drop down), and click Run.
+After running CMake, you will have the make files or Xcode project file necessary to build all of the components. Open the hifi.xcodeproj file, choose ALL_BUILD from the Product > Scheme menu (or target drop down), and click Run.
 
 If the build completes successfully, you will have built targets for all components located in the `build/${target_name}/Debug` directories.
```
Comment (yuriymacdev) [inline BUILD_OSX.md:43]: Due to a naming conflict (one of the app's libs has a name libgl which is the same as one of the OSX frameworks), running from Xcode raises runtime error (building is fine though).

File: BUILD_OSX.md [hunk 3]
```diff
@@ -17,7 +17,9 @@ Execute the `Update Shell Profile.command` script that is provided with the inst
 
 ### OSX SDK
 
-You will need the OSX SDK for building. The easiest way to get this is to install Xcode from the App Store.
+You will need version `10.12` of the OSX SDK for building. You can get that from [here](https://github.com/phracker/MacOSX-SDKs). You must copy it in to your Xcode SDK directory, e.g.
```
Comment (yuriymacdev) [inline BUILD_OSX.md:20]: Perhaps it makes sense to add here a short explanation why newer sdks fail and with what error, so the issue is searchable by those who try to build it and see the runtime crashes.

**Review by yuriymacdev — CHANGES_REQUESTED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-0e1790a.tar_374b3653.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-0e1790a_374b3653.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-0e1790a.tar_374b3653.Z

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-0e1790a_374b3653.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-0e1790a_374b3653.exe

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-bad7f35.tar_fabf8e98.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-bad7f35_fabf8e98.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-bad7f35.tar_fabf8e98.Z

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-bad7f35_fabf8e98.exe

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR653/Vircadia-Alpha-PR653-bad7f35_fabf8e98.dmg

**Comment by ArcadeFever** [comment]
Ahah! I knew the icon was in one of those .rsrc files!  Can I ask what utility you used to extract and replace the icon files with?

**Comment by kasenvr** [comment]
Alright, I opened an issue for that so we can go ahead with this and do that later. :) Thank you!

**Pull request merged**